Request/ParamConverter/PsrServerRequestParamConverter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Symfony\Bridge\PsrHttpMessage\Factory\DiactorosFactory;
+use Symfony\Bridge\PsrHttpMessage\HttpMessageFactoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Converts HttpFoundation Request to PSR-7 ServerRequest using the bridge.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PsrServerRequestParamConverter implements ParamConverterInterface
+{
+    private static $supportedTypes = [
+        'Psr\Http\Message\ServerRequestInterface' => true,
+        'Psr\Http\Message\RequestInterface' => true,
+        'Psr\Http\Message\HttpMessageInterface' => true,
+    ];
```

Comment:
We should use the `array()` syntax here. Even if you won't be able to use this annotation with PHP 5.3 or PHP 5.4, you should get a more sensible error message (the exception thrown by the `DiactorosFactory` from the bridge) instead of a parse error.